.github/workflows/if-nodejs-pr-testing.yml

```diff
@@ -72,3 +73,14 @@ jobs:
         name: Run release assets generation to make sure PR does not break it
         shell: bash
         run: npm run generate:assets --if-present
+
+      - if: steps.packagejson.outputs.exists == 'true'
+        name: Upload coverage to Codecov
+        uses: codecov/codecov-action@e28ff129e5465c2c0dcc6f003fc735cb6ae0c673
+        with:
+          fail_ci_if_error: true
+          files: ./coverage/lcov.info
+          flags: unittests
```

Comment:
What does that `flag` do? I see it being emphasized in the codecov comment? https://github.com/vishvamsinh28/website/pull/8#issuecomment-2322910083

Reply:
It labels the coverage report (total coverage of the project).

![Screenshot_20240905_213356_Chrome](https://github.com/user-attachments/assets/182a0c77-4c3f-488d-bcb6-c2d600f7edb2)

Reply:
We don't need it here. According to documentation, it is useful in case of multiple testing regimes per project. Let's remove that

Reply:
Done